docs/api/api-contract.md

```diff
@@ -0,0 +1,3 @@
+# API Contract
+
+Moved to the [Breaking Changes](breaking-changes.md) doc.
```

Comment:
I'm pretty sure theirs a way to configure redirects on the website repo so we won't need this placeholder

Reply:
> It also adds a backwards-friendly link so people still searching for "API Contract" will get a redirect. I'm not sure if this is necessary -- the actual URL of https://electronjs.org/docs/api/breaking-changes is unchanged -- would especially welcome feedback on this part.

Reply:
Oh yeah, I don't think this is required.  I think the URL being the same is enough 👍

Reply:
Sounds good to me. Literally a one-liner now.